src/main/frontend/src/App.js

```diff
@@ -0,0 +1,79 @@
+import './App.css'
+import {Container, Nav, Navbar, NavDropdown} from "react-bootstrap"
+import {Routes, Route, useNavigate, Outlet} from "react-router-dom"
+import {useEffect, useState} from "react";
+import axios from 'axios'
+import Items from "./routes/Items";
+import Detail from "./routes/Detail";
+
+function App() {
+
+    let [category, setCategory] = useState([]);
+    let navigate = useNavigate();
+
+    useEffect(() => {
+        function getItems() {
+            axios.get('/public/category')
+                .then(data => {
+                    setCategory(data.data);
+                })
+        }
+        getItems()
+    }, [])
```

Comment:
`getItems`라는 함수를 정의할거면 `useEffect`외부에 지정해주셔야합니다. 그게 아니라면 아래처럼 바꿔주세요
```suggestion
    useEffect(() => {

            axios.get('/public/category')
                .then(data => {
                    setCategory(data.data);
                })

    }, [])
```

저 함수를 `useEffect` 내부 코드블럭에 지정하게된다면 어떤 이점도 없을것으로 보이네요. 저렇게 구현하신 이유가 있으실까요?